controllers/fenceagentsremediation_controller.go

```diff
@@ -46,11 +49,14 @@ import (
 
 const (
 	// errors
-	errorMissingParams     = "nodeParameters or sharedParameters or both are missing, and they cannot be empty"
-	errorMissingNodeParams = "node parameter is required, and cannot be empty"
+	errorMissingParams             = "nodeParameters or sharedParameters or both are missing, and they cannot be empty"
+	errorMissingNodeParams         = "node parameter is required, and cannot be empty"
+	errorParamDefinedMultipleTimes = "invalid multiple definition of FAR  param"
```

Comment:
NIT: redundant space
```suggestion
	errorParamDefinedMultipleTimes = "invalid multiple definition of FAR param"
```